<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Passwordmanager</class>
 <widget class="QMainWindow" name="Passwordmanager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Менеджер паролей</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="create_password">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>161</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать пароль</string>
    </property>
   </widget>
   <widget class="QPushButton" name="view_passwords">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>160</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Посмотреть мои пароли</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>671</width>
      <height>51</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>Добро пожалоать в менеджер паролей! Тут вы можете создать новый пароль или посмтреть уже созданые</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>698</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
